pipeline/metadata/schema.py

```diff
@@ -7,8 +7,31 @@
 from dataclasses import dataclass
 from typing import Optional, List, Dict, Any, Union
 
+from apache_beam.io.gcp.internal.clients import bigquery as beam_bigquery
+
 # pylint: disable=too-many-instance-attributes
 
+SCAN_TYPE_SATELLITE = 'satellite'
+SCAN_TYPE_BLOCKPAGE = 'blockpage'
```

Comment:
I really want us to move away from the "blockpage" mentality. Also, block pages don't even make sense for the most part for HTTPS. Use "page fetch", "answer_page" or something that reflects it's about the retrieval of the domain page from the answer IP.

Reply:
I don't think it's really a mentality so much as that the file we're reading these from is called `blockpages.json`,  so it made some sense to keep using that name as we passed it around.

But done